pkg/container-utils/containerd/containerd.go

```diff
@@ -335,3 +366,10 @@ func getContainerName(container containerd.Container, labels map[string]string)
 
 	return container.ID()
 }
+
+func containerdNamespaces() []string {
+	if ns := os.Getenv(namespaces.NamespaceEnvVar); ns != "" {
```

Comment:
For later: We should start thinking about using `viper` + `cobra` to support both environment variables and flags